environs/instances/instancetype.go

```diff
@@ -26,6 +26,20 @@ type InstanceType struct {
 	Deprecated bool
 }
 
+// InstanceTypesWithCostMetadata holds an array of InstanceType and metadata
+// about their cost.
+type InstanceTypesWithCostMetadata struct {
+	// InstanceTypes holds the array of InstanceTypes affected by this cost scheme.
+	InstanceTypes []InstanceType
+	// CostUnit holds the unit in which the InstanceType.Cost is expressed.
+	CostUnit string
+	// CostCurrency holds the currency in which InstanceType.Cost is expressed.
+	CostCurrency string
+	// Divisor indicates a number that must be applied to InstanceType.Cost to obtain
+	// a number that is in CostUnit.
+	Divisor uint64
```

Comment:
I think we should document that setting (Cost)Divisor to zero will mean that the cost is treated as already being in specified unit, as the (cost-)divisor field will be omitted from the API response. i.e. the zero value is equivalent to 1.